src/components/singleProductPage/SingleProductPage.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import Slider from "react-slick";
+import "./singleProductPage.scss";
+
+export default function SingleProductPage(props) {
+  // These varaibles need to be replaced by the real data
+  const title = props.props.title;
+  const imgs = props.props.images;
+  const brand = props.props.brand;
+  const price = props.props.price;
+  const description = "Health, soft and fast running shoes";
+  const hasSize = true;
+  const sizes = [41, 42, 43, 44];
```

Comment:
so, omar has sizes too

Reply:
you mean in the add products form ?

Reply:
so we have sizes for some products that are gonna be added by the form to the db, but not all products have sizes that why we keep checking for sizes.